### PR TITLE
Add Salesforce Lightning Web Component

### PR DESCRIPTION
**Pull request by arufian**
Add Hello World! from Salesforce Lightning Web Component (LWC)

### CODE REVIEW

File: l/lwc.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <lightning-card title="HelloWorld" icon-name="custom:custom14">
+    <div class="slds-m-around_medium">
+      <p>Hello World!</p>
```
Comment (Richienb) [inline l/lwc.html:4]:
```suggestion
      <p>Hello World</p>
```

File: l/lwc.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <lightning-card title="HelloWorld" icon-name="custom:custom14">
```
Comment (Richienb) [inline l/lwc.html:2]: Do we need to set an icon or title here?

File: README.md [hunk 3]
```diff
@@ -397,6 +397,7 @@ Make sure to see [CONTRIBUTING.md](/CONTRIBUTING.md) for instructions on contrib
 * [Leszek](l/leszek.leszek)
 * [Limbo](l/limbo.b)
 * [Lingo](l/lingo.lg4)
+* [Lightning Web Component](l/lwc.html)
```
Comment (Richienb) [inline README.md:400]: Remember to also update the languages count.

**Review by Richienb — CHANGES_REQUESTED** [review]

**Comment by arufian** [comment]
@Richienb thanks for pointing out. Will fix it asap.

**Pull request merged**